servicex/templates/app/deployment.yaml

```diff
@@ -40,6 +40,10 @@ spec:
           value: "/opt/servicex/app.conf"
         - name: INSTANCE_NAME
           value: {{ .Release.Name }}
+    {{- if .Values.app.logLevel }}    
```

Comment:
Why would Log Level ever not be set? There is a default value in values.yaml... Is there a particular reason why anyone would want to leave this env var out?

Reply:
No not really, but I prefer to have a check just in case there's some way that the things get here without the loglevel being set.  E.g. the admin has a logLevel:      line without a value.

Reply:
By that logic we'd have to wrap every single environment variable with a check like this (which I don't suggest).

Reply:
I've removed the test